docs/testing/releases/490.md

```diff
@@ -0,0 +1,109 @@
+
+## Feature plugin and package inclusion in WooCommerce core
+
+### Remove deprecated shortcode save handling from SSR Blocks (#4010)
+
+Smoke test these blocks can be inserted and render correctly:
+
+* Hand-Picked products Block
+* Best Selling Products Block
+* Top Rated Products Block
+* Newest Products Block
+* On Sale Products Block
+* Products by Category Block
+* Products by Tag Block
+* Products by Attribute Block
+
+### Fix Featured Product and Featured Category button alignment (#4028)
+
+1. Add a Featured Product and Featured Category blocks.
+2. Without making any change on them, preview them in the frontend and verify the button is centered.
+3. Change the alignment of the button to the left, right, back to the center, etc. and verify it works.
+
+| Before | After |
+| --- | --- |
+| ![imatge](https://user-images.githubusercontent.com/3616980/113333925-5985a000-9323-11eb-8c11-25796187bbcc.png) | ![imatge](https://user-images.githubusercontent.com/3616980/113333839-3e1a9500-9323-11eb-9e12-4bd783487638.png) |
+
+### Enable Google Analytics Integration (#4020)
+
+- Install the [Google Analytics Integration Extension](https://github.com/woocommerce/woocommerce-google-analytics-integration)
+- Setup the extension with a G-X type ID from your GA account 
+- Install [Google Analytics Debugger](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en) extension and enable it
+- Go to the store and trigger some events using the All Products Block and Cart block (feature plugin only). See list of events in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3953.
+- The console will show when events are fired, and the Google Analytics Debugger will also show logs.
+
+To test SSR blocks:
+
+1. Insert product grid block into a page
+2. View the page. Check a view_item_list event fired in GA.
+
+To test the checkout functionality (feature plugin only):
+
+1. Visit checkout with items in your cart. You will see a begin_checkout event
+2. Start editing fields. Depending on the field you edit (which step) you will see a checkout_progress event. e.g. adding billing information will add checkout_progress with a number indicating the step you are on.
+3. Select a shipping rate. See that a set_checkout_option event fired.
+4. Select a payment method. See that a set_checkout_option fired.
+5. Leave a required field blank and place order. See _exception_ event.
+6. Place order. See add_payment_info event.
+
+## Feature plugin only
+
+### General smoke testing
+
+The following are related to various changes impacting some existing flows so just need validated there's nothing broken:
+
+-   Cart and checkout coupons interaction works as expected.
+-   Cart and checkout changing quantities works as expected.
+-   Cart and checkout shipping interaction works as expected.
```

Comment:
We should probably make it explicit we're referring to Cart and Checkout block flow testing here.

Reply:
Good point! I rewrote some of the testing steps in 0cc8945631a1954b048caeaa12a8c1c730bd6951 so they are clearer and easier to read.